featurebyte/query_graph/node/base.py

```diff
@@ -262,7 +262,16 @@ def derive_node_operation_info(
             "is_time_based": any(input_.is_time_based for input_ in inputs)
             or operation_info.is_time_based,
         }
-        return OperationStructure(**{**operation_info.dict(), **update_args})
+        constructor_args = {
+            "columns": operation_info.columns,
+            "aggregations": operation_info.aggregations,
+            "output_type": operation_info.output_type,
+            "output_category": operation_info.output_category,
+            "row_index_lineage": operation_info.row_index_lineage,
+            "is_time_based": operation_info.is_time_based,
```

Comment:
Can `is_time_based` be set as

```
any(input_.is_time_based for input_ in inputs) or operation_info.is_time_based
```

directly? That seems clearer and then `update_args` is unnecessary.

Reply:
Good point, will update this part to make it cleaner. Maybe i should pass the parameter directly to the `OperationStructure` without constructing a dictionary here.